template_macros.yaml

```diff
@@ -1,4 +1,16 @@
-latest_released_version: v2.3.0
-latest_release_name: levski
-dev_version: main
-api_version: v3
\ No newline at end of file
+# version is the git branch or tag
+#
+# Its value is used in Github URLs
+#
+# In the main branch, it should always be set to "main".
+# Upon releases and in edgex-docs release branches (e.g. levski, minnesota),
+# the value should be replaced with the git tag that is the full semantic
+# version.
+#
+# Examples: main, v2.3.0, v2.3.1, v3.0.0
```

Comment:
We are creating the minor release vanity tags, i.e. 3.0 -> latest 3.0.x,  moving forward with this upcoming release.
I have already added the 2.x vanity tags to edgex-compose

```suggestion
# Examples: main,, v3.0, v3.1
```